Index: Myuserapp/views.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- Myuserapp/views.py	(revision 1b82b44d867ade7c413a0f1aeb53a9d3c71b5423)
+++ Myuserapp/views.py	(revision a734015cd867f6f108ba8389257a10d5e17d8708)
@@ -1,19 +1,21 @@
-from django.shortcuts import render,HttpResponse,redirect
+from django.shortcuts import render,HttpResponse
 from miscellaneous import emailsend
 from authorize import authcheck
-from Myuserapp.models import UserSignup
 from Myuserapp.forms import UserSignupForm
 def index(request):
     return render(request, "home.html")
 
 # Create your views here.
 def usersignup(request):
+
     if request.method=="POST":
         email = request.POST["email"]
         form=UserSignupForm(request.POST)
         otp, time = emailsend.gen_otp()
         confirmationlink = " http://127.0.0.1:8000/verifyuser/?email=" + email + "&token=" + otp
+
         f=form.save(commit=False)
+
         f.userFullName=request.POST["name"]
         f.userEmail = request.POST["email"]
         f.userPassword = request.POST["password"]
@@ -27,23 +29,12 @@
         f.otpconfirmation=confirmationlink
         f.isActive=True
         f.isverified=False
-        f.usertoken=otp
         f.save()
         emailsend.sendmail("link",email,confirmationlink)
         return render(request,'sign.html',{"sucess":True})
+
+
     return render(request,'sign.html')
-
 def verify(request):
-    useremail=request.GET['email']
-    token=request.GET['token']
-    email=UserSignup.objects.get(userEmail=useremail)
-    if email.userEmail==useremail and token==email.usertoken:
-        isverified=True
-        email.otpcolumn=''
-        email.usertoken=''
-        email.otpconfirmation=''
-        email.save()
-        up = UserSignup(userEmail=useremail, isverified=isverified)
-        up.save(update_fields=["isverified"])
-        return HttpResponse("sucessfull")
-
+    emailid=request.GET['email']
+    link=request.GET['token']
Index: hotel/urls.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- hotel/urls.py	(revision 1b82b44d867ade7c413a0f1aeb53a9d3c71b5423)
+++ hotel/urls.py	(revision a734015cd867f6f108ba8389257a10d5e17d8708)
@@ -21,9 +21,8 @@
 
 urlpatterns = [
     url('admin/', admin.site.urls),
-    url(r'^signup/$',views.usersignup),
-    url(r'^verifyuser/$',views.verify),
-    url(r'^user/',include('Myuserapp.urls'))
+    url(r'^verifyuser/$',views.usersignup),
+url(r'^user/',include('Myuserapp.urls'))
 ]
 
 
